packages/experiments/src/components/Tile/Tile.tsx

```diff
@@ -207,7 +207,7 @@ export class Tile extends React.Component<ITileProps, ITileState> {
         href={href}
         onClick={onClick}
         ref={this.props.linkRef}
-        data-selection-invoke={isInvokable && selectionIndex > -1 ? true : undefined}
+        data-selection-invoke={isSelectable || (isInvokable && selectionIndex > -1) ? true : undefined}
```

Comment:
It's possible to select items but not be able to invoke them. We need `isInvokable` to be enforced, still.